src/main/java/org/elasticsearch/index/query/FuzzyQueryBuilder.java

```diff
@@ -62,6 +62,14 @@ public FuzzyQueryBuilder(String name, Object value) {
     }
 
     /**
+     * package private constructor for serialization only
+     */
+    FuzzyQueryBuilder() {
+        this.name = null;
+        this.value = null;
```

Comment:
yea constructors would become private, which is good, you sure constants would need to become public? or is it package private enough?